### PR TITLE
add banner for Grace Hopper Celebration Open Source Day

### PR DESCRIPTION
**Pull request by Trott**

### CODE REVIEW

**Comment by Trott** [comment]
Dark mode screenshot:

image

**Comment by Trott** [comment]
Light mode:

image

**Comment by nschonni** [comment]
LGTM, but the banner is a little big at ~2MB. Is it possible to hotlink it from the source or optimize it a bit?

**Comment by Trott** [comment]
> LGTM, but the banner is a little big at ~2MB. Is it possible to hotlink it from the source or optimize it a bit?

Oops, yeah, I meant to switch it to a jpeg or at least shrink down the png a bit and forgot. Sorry! I'll do that later.

I didn't want to hot link because the source banner is huge and dominates the layout:

image

So I trimmed the top and bottom to make it fit in with the site a bit better.

**Comment by nschonni** [comment]
OK, can't think if there would be a way to source the original, and clip with CSS, but might be not worth the effort

**Comment by Trott** [comment]
OK, the png file is now a lot smaller. No noticeable change in the way it looks in the browser, at least as far as I can tell. Here are screenshots:

image

image

**Review by nschonni — APPROVED** [review]

**Comment by nschonni** [comment]
FYI, I think the .dev site is trying to pull in these files, so I'm expecting it might choke after this change

**Review by fhinkel — APPROVED** [review]

**Review by aymen94 — APPROVED** [review]

**Review by tuanggolt — COMMENTED** [review]
open

**Comment by Trott** [comment]
> open

Spam user that seems to comment "open" on everything is now blocked.

**Comment by ovflowd** [comment]
Hey @Trott sorry for necro-bumping here. But could you change the line where is `/static/` to include `https://nodejs.org`? This would allow us from Nodejs.dev to pull this image without copying it.

Thanks!

File: locale/en/site.json [hunk 1]
```diff
@@ -159,10 +159,11 @@
...
   "banners": {
     "index": {
-      "startDate": "2022-06-27T16:00:00.000Z",
-      "endDate": "2022-07-13T16:00:00.000Z",
-      "text": "New security releases now available for Node.js 14, 16, and 18 release lines",
-      "link": "https://nodejs.org/en/blog/vulnerability/july-2022-security-releases/"
+      "startDate": "2022-07-29T16:00:00.000Z",
+      "endDate": "2022-08-11T16:00:00.000Z",
+      "text": null,
+      "html": "<img alt='Grace Hopper Celebration' src='/static/images/ghc-banner.png' />",
```
Comment (ovflowd) [inline locale/en/site.json:165]: TL;DR:

```js
 "html": "<img alt='Grace Hopper Celebration' src='https://nodejs.org/static/images/ghc-banner.png' />",
```

**Comment by ovflowd** [comment]
PR on Nodejs.dev: https://github.com/nodejs/nodejs.dev/pull/2618

File: locale/en/site.json (hunk 1 above)
Comment (Trott) [reply locale/en/site.json:165]: Can you open the PR for that?

Comment (ovflowd) [reply locale/en/site.json:165]: Here we go: https://github.com/nodejs/nodejs.org/pull/4762

**Pull request merged**